zipkin-collector/scribe/src/main/java/zipkin2/collector/scribe/NettyScribeServer.java

```diff
@@ -61,6 +60,7 @@ void start() {
 
   void close() {
     if (channel == null) return;
+    // TODO: chain these futures, and probably block a bit
```

Comment:
this is the only one I left as a compile warning, as I think it is legit. help wanted on the most elegant way to chain this granted the return val is void